docs/snippets/common/component-story-custom-args-mapping.js.mdx

```diff
@@ -0,0 +1,27 @@
+```js
+// YourComponent.stories.js
+
+import startCase from 'lodash/startCase';
+
+import { YourComponent } from './your-component';
+import * as icons from './icons';
```

Comment:
We should show the explicit value here and not leave it up to the reader's imagination

Reply:
Done